src/lib/pipeline/m2/index.js

```diff
@@ -111,7 +120,9 @@ class M2 extends THREE.Group {
 
       geometry.faceVertexUvs = [uvs];
 
-      const mesh = new Submesh(id, geometry, textureUnits);
+      const isBillboard = indexedBones[submesh.rootBone].userData.isBillboard === true;
+
+      const mesh = new Submesh(id, geometry, textureUnits, isBillboard);
```

Comment:
I just realized we're not passing the `submesh` data-blob to Submesh. Perhaps we should?